sdks/python/apache_beam/runners/worker/bundle_processor.py

```diff
@@ -819,7 +819,7 @@ def __init__(self,
     # There is no guarantee that the runner only set
     # timer_api_service_descriptor when having timers. So this field cannot be
     # used as an indicator of timers.
-    if self.process_bundle_descriptor.timer_api_service_descriptor:
+    if self.process_bundle_descriptor.timer_api_service_descriptor.url:
```

Comment:
Is it possible for `timer_api_service_descriptor` to be `None` as well?

Reply:
I believe for proto3 all fields are optional and have a default instance thus can't be None.